docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -104,7 +104,7 @@ It is normal for the _"Restart_As_Admin"_ button to appear untoggled. This is al
 The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for DDS format skins (the skins with the numbered folders in them), which are commonly used for Vanilla Titanfall 2, with Northstar generally opting to convert them to mods.
 
 This format is older and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)).
-It can still be used on Vanilla by making use of a [vanilla profile](../../using-northstar/advanced.md#vanilla-profile).
+It can still be used on Vanilla by making use of a [vanilla via Northstar](../../using-northstar/vanilla.md#vanilla-with-mods).
```

Comment:
This sentence doesn't sound quite right ^^"

> [...] by making use of a vanilla via Northstar.

Reply:
Ah, I see this now, I meant to say `by making use of vanilla via Northstar`, though this could still probably have better wording